reference/docs-conceptual/PSScriptAnalyzer/rules-recommendations.md

```diff
@@ -0,0 +1,208 @@
+---
+description: This article lists best-practice recommendations and the rules associated with them.
+ms.date: 03/22/2022
+title: PSScriptAnalyzer rules and recommendations
+---
+# PSScriptAnalyzer rules and recommendations
+
+The following guidelines come from a combined effort from both the PowerShell team and the
+community. The guidelines are organized by type. Within each type there is a list of rules. The
+rules are group by the **Severity** defined in the implementation of the **PSScriptAnalyzer** rule.
+The severity level labeled as 'TBD' means "To be determined". These are recommendations that do not
+currently have rules defined.
+
+## Cmdlet Design Rules
+
+### Severity: Error
+
+No rules defined.
+
+### Severity: Warning
+
+- Use Only Approved Verbs [UseApprovedVerbs](Rules/UseApprovedVerbs.md)
+- Cmdlets Names: Characters that cannot be Used
+  [AvoidReservedCharInCmdlet](Rules/ReservedCmdletChar.md)
+- Parameter Names that cannot be Used
+  [AvoidReservedParams](Rules/ReservedParams.md)
+- Support Confirmation Requests
+  [UseShouldProcessForStateChangingFunctions](Rules/UseShouldProcessForStateChangingFunctions.md)
+  and
+  [UseShouldProcessForStateChangingFunctions](Rules/UseShouldProcessForStateChangingFunctions.md)
+- Must call ShouldProcess when ShouldProcess attribute is present and vice
+  versa.[UseShouldProcess](Rules/ShouldProcess.md)
+- Nouns should be singular
+  [UseSingularNouns](Rules/UseSingularNouns.md)
+- Module Manifest Fields
+  [MissingModuleManifestField](Rules/MissingModuleManifestField.md)
+  - Version
+  - Author
+  - Description
+  - LicenseUri (for PowerShell Gallery)
+- Switch parameters should not default to true
+  [AvoidDefaultValueSwitchParameter](Rules/AvoidDefaultValueSwitchParameter.md)
+
+### Severity: Information
+
+No rules defined.
+
+### Severity: TBD
+
+- Support Force Parameter for Interactive Session
+- If your cmdlet is used interactively, always provide a Force parameter to override the interactive
+  actions, such as prompts or reading lines of input). This is important because it allows your
+  cmdlet to be used in non-interactive scripts and hosts. The following methods can be implemented
+  by an interactive host.
```

Comment:
Should this be a sub-bullet of the entry above or part of it? These seem to go together.